tools/fpgabist/bist_common.py

```diff
@@ -32,9 +33,21 @@
 
 # TODO: Use AFU IDs vs. names of AFUs
 BIST_MODES = ['bist_afu', 'dma_afu', 'nlb_mode_3']
+REQ_CMDS = ['lspci', 'fpgainfo', 'fpgaconf', 'fpgadiag', 'fpga_dma_test',
+            'bist_app']
 
 
 # Return a list of all available bus numbers
+def check_commands():
+    terminate = False
+    for cmd in REQ_CMDS:
+        executable = find_executable(cmd, path=os.environ['PATH'])
+        if executable == cmd:
```

Comment:
From offline discussion I know what you're doing here.
Instead of using such thing, you could do:
```
path = os.environ['PATH'].split(os.pathsep)
# Remove current directory from PATH
del path[-1]
executable = find_executable(cmd, path=path)
```

Reply:
This doesn't seem to remove the cwd from the PATH. When I made this fix and tested I'm still running into the same issue. going to forego using the find_executable method because its making my life much harder than it needs to be IMO